README.md

```diff
@@ -150,26 +155,47 @@ These parameters can also be set via the appSettings.json file located in the sa
 ### Example output
 
```

Comment:
Can we just add information that `JsonProperty`'s attribute value is being generated from element codename and  `KontentElementId` attribute value is element's ID.

```suggestion

> `JsonProperty`'s attribute value is being generated from element codename (not from the type) and  `KontentElementId` attribute value is element's ID.

```